js/main.js

```diff
@@ -23,10 +23,204 @@ function getRandomNumber (minValue, maxValue) {
 
 getRandomNumber();
 
-
 // Функция для проверки максимальной длины строки.
 
-function checkStringLength (string, maxLength) {
-  return (string.length <= maxLength);
+function checkStringLength (string, length) {
+  return string.length <= length;
+}
+checkStringLength('word', 3);
+
+//Создаем 25 объектов по ТЗ из раздела 4.15
+//Указываем необходимое количество фото-объектов
+const photoObjectsNumber = 25;
```

Comment:
Это константа, пишется в верхнем регистре
```suggestion
const PHOTOS_COUNT= 25;
```